_maps/RandomRuins/BeachRuins/beach_ancient_ruin.dmm

```diff
@@ -374,6 +374,7 @@
 	},
 /obj/item/gun/energy/lasercannon{
 	pixel_y = 5;
+	
```

Comment:
```suggestion

```
committing this doesn't... actually do anything, not sure what to do so im trying this